apis/Google.LongRunning/Google.LongRunning.Snippets/OperationsClientSnippets.g.cs

```diff
@@ -53,9 +53,9 @@ public class GeneratedOperationsClientSnippets
             // Create client
             OperationsClient operationsClient = OperationsClient.Create();
             // Initialize request argument(s)
-            string formattedName = OperationsClient.FormatOperationPathName("[OPERATION_PATH]");
+            string name = "";
             // Make the request
-            Operation response = await operationsClient.GetOperationAsync(formattedName);
+            Operation response = await operationsClient.GetOperationAsync(name);
```

Comment:
The parameter name is just "name" for the method itself. I'm not sure what determines the name of the local variable to use in the snippets.